Archive.md

```diff
@@ -0,0 +1,56 @@
+# Sprint Archive
+Welcome to the Sprint Archive, where you can access all the previously assigned questions and solutions to them. Solutions become available three days after the Sprint questions are shared.
+| Edition     | Difficulty | Question Name                                           | Question Link                                                                                          | Editorial Link                                                                                                                                                                       | Video |
+|-------------|------------|---------------------------------------------------------|--------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------|
+| 1st Sprint  | Easy       | Maximum Units on a Truck                                | [Question Link](https://leetcode.com/problems/maximum-units-on-a-truck)                                | [Editorial Link](https://github.com/ituacm/ITU-ACM-22-Summer-Algorithm-Team-Bootcamp/blob/main/Regular-Question-Answers/maximum-units-on-a-truck.cpp)                                |       |

```

Comment:
Bunun yerine bence şöyle bir tablo yapsak daha hoş olur
Main olarak iki column header bulunur. Bunlar Easy ve Medium.
Bunların altında subheader olarak Question Name/Link/Editorial/Video bulunur

Her bir row için de sadece 1,2,3... sıralı olarak gider.
![image](https://user-images.githubusercontent.com/55850855/192518319-81a0c9ba-de64-4d20-bb93-e0265f89bc55.png)
Bunu örnek olarak baz alabilirsin